api/src/pcapi/routes/backoffice/templates/individual_bookings/list_rows.html

```diff
@@ -13,7 +13,7 @@
              class="form-check-input"
              name="pc-table-multi-select-check-{{ booking.id }}"
              data-id="{{ booking.id }}"
-             data-value="{{ booking.amount }}" />
+             data-value="{{ booking.quantity * booking.amount }}" />
```

Comment:
Il y a une propriété pour ne jamais se tromper : 
```suggestion
             data-value="{{ booking.total_amount }}" />
```